src/api/Cargo.toml

```diff
@@ -22,15 +22,18 @@ actix-web-httpauth = "0.8.0"
 aes-gcm = "0.10.1"
 anyhow.workspace = true
 async-trait.workspace = true
-as-types = { git = "https://github.com/confidential-containers/attestation-service.git", tag = "v0.7.0"}
-attestation-service = { git = "https://github.com/confidential-containers/attestation-service.git", default-features = false, tag = "v0.7.0", optional = true}
+# TODO: change it to "0.8.0", once released.
+as-types = { git = "https://github.com/BaoshunFang/attestation-service.git"}
+# TODO: change it to "0.8.0", once released.
+attestation-service = { git = "https://github.com/BaoshunFang/attestation-service.git", default-features = false, optional = true}
```

Comment:
@BaoshunFang Now your AS PR is merged, we can keep this dependence upstream :)

Reply:
Done